tests/core/pyspec/eth2spec/test/fulu/sanity/test_lookahead_slots.py

```diff
@@ -59,7 +59,7 @@ def test_effective_decrease_balance_updates_lookahead(spec, state):
     actual_lookahead = simulate_lookahead(spec, state)[: spec.SLOTS_PER_EPOCH]
 
     if not is_post_fulu(spec):
-        # Pre-EIP-7917, effective balance changes changes the next epoch's lookahead
+        # Pre-EIP-7917, effective balance changes the next epoch's lookahead
```

Comment:
Hi @goldlinker the original is technically correct. Though I see how this could be confusing.

*Changes* to effective balances *changes* the next epoch's lookahead.